recipes/assimp/5.x/conanfile.py

```diff
@@ -298,6 +318,8 @@ def _patch_sources(self):
         ]:
             save(self, os.path.join(self.source_folder, "contrib", contrib_header),
                  f"#include <{include}>\n")
+        if Version(self.version) >= "5.4.0":
+            rmdir(self, self.source_path.joinpath("contrib", "utf8cpp"))
```

Comment:
This seems to only be used in samples and there's a change in the release mentioning this, but why is it only now removed? (I might have checked the wrong diff now that I think about it!)

Reply:
Oh, it's probably unnecessary in that case then. Let me see if it builds ok without this...

Reply:
It was necessary. :slightly_smiling_face:

Reply:
Thanks for double-checking :)